app/views/categories/index.html.erb

```diff
@@ -1,5 +1,5 @@
 <ul>
   <% @categories.each do |cat| %>
-    <li><%= cat.name %></li>
+    <li><%= link_to cat.name, items_path(:category => cat.name) %></li>
```

Comment:
Think about this path helper, if it is an index path it shouldn't take in an argument. If it's a path helper for a show page you should pass in the category.